content/en/platform/corda/5.0/glossaryitems.json

```
@@ -1,37 +1,262 @@
-        "definition": "Corda Package Bundle. These are complete applications minus the “run time information” required to onboard entities.",
-        "more": ""
+        "definition": "Corda Package Bundle. A signed ZIP/JAR collection of CPKs that forms a complete application suite and contains all the code that a virtual node must operate, minus the specific network details. ",
+        "more": "This separation is present in Corda 5 so that software can be developed and tested against development networks and then fuller UAT networks before final publication of the production CPI with the surety that the code has not changed. For more information, see <a href=\"../developing-applications/packaging.html\">Packaging</a>."
     },
     "CPI": {
-        "definition": "Corda Package Installer. These contain the CPB and all of the information required to join and participate in an application network.",
-        "more": "The only difference between a development CPI and a production CPI is the network information. Both environments deploy the same CPB but that CPB will be wrapped into a different CPI for installation and setup."
-    }    
+        "definition": "Corda Package Installer. A signed ZIP/JAR combination of a CPB and a Group Policy File that defines not only the application code that a virtual node will run, but also the details of the MGM with which to register, and the details of network PKI requirements.",
+        "more": "A CPI must be installed into a Corda Cluster before creating and registering a virtual node, as it defines the application software that will be run and the requirements for membership on that network. For more information, see <a href=\"../developing-applications/packaging.html\">Packaging</a>.​"
+    },   
+    "Autoscaling": {
+        "definition": "A technique used to automatically adjust the amount of computing resources allocated to an application or service based on its workload.",
+        "more": "If a sudden increase in demand is experienced for an application or service, autoscaling increases provided resources to cope with demand. This intervention can be manual or automatic and is designed to reduce chances of service interruption, or performance issues."
+    }, 
+    "Bootstrapping": {
+        "definition": "To boot or to load a program using a much smaller initial program to load in the desired program.",
+        "more": "This is usually an operating system or setup files that can be implemented automatically or manually."
+    }, 
+    "CA": {
+        "definition": "Certificate Authority. The holders of a PKI trust root that can issue certificates to customers." ,
+        "more": "When you set up a membership group, you must decide which certificate authorities apply to various identities: TLS, ledger, and REST user identities."
+    }, 
+    "Cluster": {
+        "definition": "A complete set of worker processes. Clusters require a fully-functioning virtual node infrastructure",
```

Does fully functioning get a hyphen?
```suggestion
        "definition": "A complete set of worker processes. Clusters require a fully functioning virtual node infrastructure".
```